primitives/src/xvm.rs

```diff
@@ -102,11 +102,13 @@
     /// - `source`: Caller Id.
     /// - `target`: Target contract address.
     /// - `input`: call input data.
+    /// - `value`: value to transfer.
     fn call(
         context: Context,
         vm_id: VmId,
         source: AccountId,
         target: Vec<u8>,
         input: Vec<u8>,
+        value: Balance,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new field `value` of type `Balance` has been added to the `call` function parameters. Ensure that this change is properly documented and communicated to users of the function.
```diff
     /// - `source`: Caller Id.
     /// - `target`: Target contract address.
     /// - `input`: call input data.
+    /// - `value`: value to transfer.
     fn call(
         context: Context,
         vm_id: VmId,
         source: AccountId,
         target: Vec<u8>,
         input: Vec<u8>,
+        value: Balance,
     ) -> CallResult;
 }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->